content/news/2021.pml

```diff
@@ -380,7 +380,7 @@ Older News: <%=reflist%>
 
    On 6 August <a href="https://www.darpa.mil">DARPA</a> brought the
    &ldquo;SMACCMcopter&rdquo;
-   to <a href="https://loonwerks.com/publications/cofer2021defcon.html">DEF&nbsp;CON</a>
+   to <a href="https://www.defcon.org/">DEF&nbsp;CON</a>
```

Comment:
Would prefer the old more specific link to `https://defcon.org/html/defcon-29/dc-29-index.html` so the year matches up.